### PR TITLE
feat add direct load for dateset

### LINKED ISSUE #154: [Feature]:  Direct dataset loading
### Contact Details(optional)

_No response_

### What feature are you requesting?

We have two ways of loading dataset for now. 
1,  load plain text of one specific field and the embedding of another field (or same field )
2,  load strings consist of  whole field with corresponding titles and the embedding of the string.

In the above ways of loading dataset , the embeddings are calculated within loading process,  this slow down the speed of loading.  The testing and other purpose,  some dataset contains the embedding  field.  We should have a method of directly load dataset without run encode process. 

Create an api or update the existing one (POST /train)

### PR DESCRIPTION
**Pull request by Micost**
**Description**

This PR fixes #154
Update load dataset api by enable directly load embeddings from dataset,  no external tokenization needed 

**Notes for Reviewers**

**[Signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: backend/src/repository/rag/chat.py [hunk 1]
```diff
@@ -58,20 +58,51 @@ async def load_csv_file(self, file_name: str, model_name: str) -> bool:
...
+        if param.directLoad:
+            self.load_data_set_directly(param=param)
+        elif param.embedField is None or param.resField is None:
             self.load_data_set_all_field(dataset_name=param.dataSet) 
         else:
-            self.load_data_set_by_field(param=param)   
+            self.load_data_set_by_field(param=param)
         return True
 
+    def load_data_set_directly(self, param: TrainFileIn)->bool:
```
Comment (Aisuko) [inline backend/src/repository/rag/chat.py:77]:
```suggestion
    def load_data_set_directly(self, param: TrainFileIn)->bool:
        r"""
        We load the the .csv file directly
        
        @param param: the instance of TrainFileIn
        
        @return: boolean
        """
```

**Review by Aisuko — APPROVED** [review]

**Review by Aisuko — APPROVED** [review]
LGTM

**Comment by Aisuko** [comment]
@Micost check the change I made. And we can merge this

**Comment by Micost** [comment]
> @Micost check the change I made. And we can merge this

I've update with new comment, this api is used for dataset which already contains embeddings

**Review by Aisuko — COMMENTED** [review]
approve

**Review by Aisuko — APPROVED** [review]

**Pull request merged**